playbook-website/package.json

```diff
@@ -9,9 +9,12 @@
   },
   "dependencies": {
     "@rails/webpacker": "5.2.1",
+    "leaflet": "1.7.1",
     "playbook-ui": ">= 1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-leaflet": "3.1.0",
```

Comment:
Anything above 3.1.0 breaks the map because of JS parsing errors coming from react-leaflet